src/Parser/TokenIterator.php

```diff
@@ -160,8 +161,10 @@ public function getSkippedHorizontalWhiteSpaceIfAny(): string
 	/** @phpstan-impure */
 	public function joinUntil(int ...$tokenType): string
 	{
+		$tokenTypeMap = array_flip($tokenType);
```

Comment:
is this faster then `array_fill_keys($tokenType, true)`?

Reply:
It's indistinguishable so changing to `array_fill_keys()` is fine by me :)